lib/player.js

```diff
@@ -685,6 +703,68 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     if (mediaElement) {
       this.attach(mediaElement, /* initializeMediaSource= */ true);
     }
+
+    if (canvas) {
+      this.canvas_ = canvas;
+    }
+  }
+
+  /**
+   * Create a shaka.lcevc.Dil object
+   * @private
+   */
+  createDIL_() {
+    if (this.lcevcDil_ == null) {
+      this.lcevcDil_ = new shaka.lcevc.Dil(
+          /** @type {HTMLVideoElement} */ (this.video_),
+          this.canvas_,
+          this.lcevcDilConfig_,
+      );
+      if (this.mediaSourceEngine_) {
+        this.mediaSourceEngine_.updateLcevcDil(this.lcevcDil_);
+      }
+    }
+  }
+
+  /**
+   * Close a shaka.lcevc.Dil object
+   * @private
+   */
+  closeDIL_() {
+    if (this.lcevcDil_ != null) {
+      this.lcevcDil_.hideCanvas();
```

Comment:
The `release` method already hides the canvas, so you don't need to call this explicitly.

Reply:
Removed hideCanvas call from closeDil